src/logging_solver.cpp

```diff
@@ -476,7 +497,11 @@ void LoggingSolver::assert_formula(const Term & t)
   wrapped_solver->assert_formula(lt->wrapped_term);
 }
 
-Result LoggingSolver::check_sat() { return wrapped_solver->check_sat(); }
+Result LoggingSolver::check_sat()
+{
+  Result r = wrapped_solver->check_sat();
```

Comment:
return directly

Reply:
Addressed in e3e74f36278132d722bf3991c6515fa38de16c1b